android/src/main/java/io/customer/reactnative/sdk/CustomerIOReactNativeModule.kt

```diff
@@ -82,8 +82,16 @@ class NativeCustomerIOModule(
     }
 
     @ReactMethod
-    fun identify(identifier: String, attributes: ReadableMap?) {
-        customerIO()?.identify(identifier, attributes.toMap())
+    fun identify(identifier: String?, attributes: ReadableMap?) {
+        if (identifier == null && attributes == null) {
+            logger.error("Please provide either an ID or traits to identify.")
+            return
+        }
+        identifier?.let {
+            customerIO()?.identify(identifier, attributes.toMap())
```

Comment:
`attributes` is nullable here, shouldn't it be `attributes?.toMap()` or does `toMap()` handle null values and if so what does it return?

Reply:
Yes it is being handled by [toMap()](https://github.com/customerio/customerio-reactnative/blob/de2852ea840ba6f06bd01bc83c8a874385832dbd/android/src/main/java/io/customer/reactnative/sdk/extension/TypeConversion.kt#L8-L10) and returns EmptyMap if `attributes` is null.